src/ApiService/ApiService/QueueSignalREvents.cs

```diff
@@ -9,8 +9,8 @@ public class QueueSignalREvents {
         _loggerFactory = loggerFactory;
     }
 
-    //[Function("QueueSignalREvents")]
-    //[SignalROutput(HubName = "dashboard")]
+    [Function("QueueSignalREvents")]
```

Comment:
If we are enabling it here, where is the disabling of the python instance?

Reply:
He renamed the config file of the python function

Reply:
What Cheick said it right. The change isn't super obvious in github's diff UI but this link should highlight it: https://github.com/microsoft/onefuzz/pull/1927/files#diff-db9d09ebdbe821d0d51f0db1a6a8ac62b20d1c749944c030f82259d30a773908